wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemram/SystemRamInfo.java

```diff
@@ -51,4 +51,35 @@ public long getSystemRamUsed() {
     public boolean isSystemRamLimitExceeded() {
         return isSystemRamLimitExceeded;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        SystemRamInfo that = (SystemRamInfo)o;
+
+        if (systemRamUsed != that.systemRamUsed) return false;
+        if (systemRamTotal != that.systemRamTotal) return false;
+        return isSystemRamLimitExceeded == that.isSystemRamLimitExceeded;
+
+    }
+
+    @Override
+    public int hashCode() {
+        int result = (int)(systemRamUsed ^ (systemRamUsed >>> 32));
```

Comment:
please, use `Long.hashCode` or `Objects.hash()` or [check this out](https://github.com/evoevodin/intellij-idea-tools/tree/master/templates/equals-hashcode/collections-never-null)